app/routes/users.js

```diff
@@ -6,20 +6,7 @@ const usersController = require('../controllers/users');
 
 router.post('/register', usersController.register);
 router.post('/login', usersController.login);
-
-router.post('/confirmToken', async (req, res) => {
-  const { username, token } = req.body;
-  if (username && token) {
-    const result = await auth.authorizeUser(username, token);
-    if (result.success) {
-      res.status(200).json({ username });
-    } else {
-      res.status(401).json({ error: result.message });
-    }
-  } else {
-    res.status(401).json({ error: 'User is not authenticated!' });
-  }
-});
+router.post('/confirmToken', usersController.confirmToken);
```

Comment:
BE people don't like camelCase for URLs, it's kinda frowned upon and a common tell from FE people designing APIs haha.

It's recommended to use hyphens because it's a non-word character.

Reply:
One example of Google recommending doing it like that: https://support.google.com/webmasters/answer/76329?hl=en